app/src/main/java/com/poupa/vinylmusicplayer/preferences/SongConfirmationPreference.java

```diff
@@ -0,0 +1,40 @@
+package com.poupa.vinylmusicplayer.preferences;
+
+
+import java.util.Arrays;
+import java.util.List;
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+import com.kabouzeid.appthemehelper.common.prefs.supportv7.ATEDialogPreference;
+import com.poupa.vinylmusicplayer.R;
+import com.poupa.vinylmusicplayer.dialogs.BottomSheetDialog.BottomSheetDialogWithButtons;
+import com.poupa.vinylmusicplayer.dialogs.BottomSheetDialog.BottomSheetDialogWithButtons.ButtonInfo;
+import com.poupa.vinylmusicplayer.util.PreferenceUtil;
+
+
+public class SongConfirmationPreference extends ATEDialogPreference {
+   public static BottomSheetDialogWithButtons.ButtonInfo ASK = new ButtonInfo(PreferenceUtil.ENQUEUE_SONGS_CHOICE_ASK, R.string.action_always_ask_for_confirmation, R.drawable.ic_close_white_24dp, null);
```

Comment:
I think a list icon would fit better than `ic_close_white_24dp`.

Reply:
Better like that ?

Reply:
I would have picked 
![image](https://user-images.githubusercontent.com/73739153/222968603-08d2d2ae-a038-4885-babf-1e3e51d9d6f9.png)
but I don't know if that icon is available.

Reply:
Don't really want to add a new icon in the app for this

Reply:
I don't really see the problem with adding a new icon but yeah the difference is only minor and not that important, the playlist icon is certainly better than the close icon.